.cake-scripts/version.cake

```diff
@@ -1,3 +1,5 @@
+#addin nuget:file:../.cake-tools/Addins/?package=Cake.Git
```

Comment:
Thanks, LGTM. I am just curious why do we need this change? Is there any documentation? Best practice?

Reply:
I found that without that `addin` directive the line
``` csharp
context.GitBranchCurrent(".");
```
shows as an intellisense error for me using vs code with all the prerequisite extensions installed (I guess its because within the context of that `version.cake` file it doesn't have information on that referenced library as the `Cake.Git` addin is propagated down from `build.cake`) - i.e. this is just to improve the developer experience and silence "false alarm" intellisense errors

Cake script build practices were hard to come by unfortunately :( but the main reason I used the "local reference" format for the addin reference was to prevent potential errors resulting from having multiple different versions of the same package reference when a developer forgets to keep them consistent etc.

Reply:
Ok, thanks. I am not using IntelliSense for Cake, that is why I was not aware of the "error" message. I think that should be fixed by the plugin. We can keep it, but the CLI complains about the missing version:

    The 'addin' directive is attempting to install the 'Cake.Git' package without specifying a package version number

I think we should change it to `#addin nuget:file:../.cake-tools/Addins/?package=Cake.Git&version=2.0.0`?